lib/active_shipping/carrier.rb

```diff
@@ -92,6 +92,16 @@ def find_tracking_info(tracking_number, options = {})
       raise NotImplementedError, "#find_tracking_info is not supported by #{self.class.name}."
     end
 
+    # Get a list of services available for the a specific route
+    #
+    # @param origin_country [String] The country of origin
```

Comment:
1. Can we be explicit in the argument this should be the country code as opposed to the country name. 

e.g. `@param origin_country_code ...`

2. Furthermore call it out as the ISO-alpha 2 which is available from `ActiveUtils`

e.g. `[String] The ISO alpha 2 origin country code`

IMO no. 1 would at least add clarity if they should pass `CA` or `Canada` and not have to deal with doing a look up to convert one to the other.